### ISSUE TITLE
Fix SDS and COA count in sitemap

### ISSUE THREAD

**Issue by MeghnaTewari** [hlxsites/moleculardevices#1085]
SDS and COA count in the sitemap is very low, it shows 32. However, it is fine in the Coveo search.

Coveo
![image](https://github.com/hlxsites/moleculardevices/assets/138755144/df09606c-50fa-4208-aef5-e247040c2c19)

Sitemap- 
https://www.moleculardevices.com/files-sitemap.xml
![image](https://github.com/hlxsites/moleculardevices/assets/138755144/84027531-64eb-4c75-8740-51a5d385ad3d)

**Comment by mhaack** [comment]
@jose-correia can you check:
a) do we have them
b) are these published

Only published are in the site map. While index is based on the query-sheet files tab.

**Comment by mhaack** [comment]
@MeghnaTewari the SDS docs for example contain all languages/regions (see https://www.moleculardevices.com/search-results#t=All&sort=relevancy&f:@md_contenttype=[SDS]) is this expected?

**Comment by mhaack** [comment]
@MeghnaTewari additionally on https://live-md30.pantheonsite.io/search-results#t=All&sort=relevancy I don't even see SDS docs? Should these even been in Coveo?

**Comment by MeghnaTewari** [comment]
> @MeghnaTewari the SDS docs for example contain all languages/regions (see [https://www.moleculardevices.com/search-results#t=All&sort=relevancy&f:@md_contenttype=[SDS]](https://www.moleculardevices.com/search-results#t=All&sort=relevancy&f:@md_contenttype=%5BSDS%5D)) is this expected? Or EN only.

@mhaack  Yes, this is expected. The count for SDS and COA which is coming in coveo should be as it is on the sitemap.

**Comment by MeghnaTewari** [comment]
> @MeghnaTewari additionally on https://live-md30.pantheonsite.io/search-results#t=All&sort=relevancy I don't even see SDS docs? Should these even been in Coveo?

@mhaack It is there on https://live-md30.pantheonsite.io/search-results#t=All&sort=relevancy
Please click show more twice.

![image](https://github.com/hlxsites/moleculardevices/assets/138755144/1e64f4f3-e084-4504-8fb5-bc4b3a43111c)

**Comment by mhaack** [comment]
Ah thanks, only clicked 1 time on "show more"

**Comment by andreituicu** [comment]
@mhaack you can also view them in the coveo xml: https://moleculardevices.com/coveo/coveo-xml.xml
which comes from https://github.com/hlxsites/moleculardevices/blob/main/coveo/coveo-xml.xml

**Comment by MeghnaTewari** [comment]
@mhaack Looks good now.
